server/rooms.ts

```diff
@@ -934,9 +938,44 @@ export abstract class BasicRoom {
 			this.reportJoin('l', user.getIdentity(this.roomid), user);
 		}
 		if (this.game && this.game.onLeave) this.game.onLeave(user);
+
+
 		return true;
 	}
 
+	runAutoModchat() {
+		if (!this.settings.autoModchat || this.settings.autoModchat.active) return;
+		// they are staff and online
+		const staff = Object.values(this.users).filter(u => this.auth.isStaff(u.id));
+		if (!staff.length && this.settings.autoModchat) {
+			const {rank, time} = this.settings.autoModchat;
+			this.modchatTimer = setTimeout(() => {
+				this.settings.modchat = rank;
+				this.add(
+					`|raw|<div class="broadcast-blue"><b>This room has had no active staff for ${Chat.toDurationString(time)},` +
```

Comment:
Use  < strong > instead of < b >. Also I'd drop the exclamation mark, since I don't think automodchat is _that_ exciting :P